cclib/parser/orcaparser.py

```diff
@@ -403,6 +403,25 @@ def splitter(line):
             if not hasattr(self, 'grads'):
                 self.grads = []
             self.grads.append(grads)
+        
+        # Grab MP2 gradients
```

Comment:
This involves a little bit of code duplication. We could fix that by doing the following.

The `next` statements at 392 and 393 aren't rewriting `line` so at 394 we could add `if line[:18] == 'CARTESIAN GRADIENT' or line[:22] == 'The final MP2 gradient':` 

Then lines 413-424 could be removed.

Remember to take the comment here on 407 the comment and combine it with the comment on 390

Let me know if this makes sense

Reply:
Yes, that makes sense for the most part.

I combined the two signals for cartesian gradient and MP2 gradient and merged the common code. Let me know if this is what you were looking for, or what else you would like me to correct.